constants.js

```diff
@@ -115,6 +115,7 @@ const constants = {
     legacyLocations: ['sproxyd', 'legacy'],
     /* eslint-disable camelcase */
     externalBackends: { aws_s3: true, azure: true, gcp: true },
+    replicationBackends: ['aws_s3', 'azure', 'gcp'],
```

Comment:
I might prefer the same construct as for the ```externalBackends``` constant

Reply:
Good call. Updated!